gulp-tasks/build.js

```diff
@@ -1,37 +1,55 @@
-var gulp = require('gulp'),
-    minCSS = require('gulp-clean-css'),
-    jshint = require('gulp-jshint'),
-    stylish = require('jshint-stylish'),
-    rev = require('gulp-rev'),
-    uglify = require('gulp-uglify'),
-    del = require('del');
-    usemin = require('gulp-usemin')
+var gulp = require('gulp');
+var useref = require('gulp-useref');
```

Comment:
is not better to not write `var` ? is less code.  I will look into rarefaction more : we can maybe use something like `require("gulp-load-plugins")`